types/web3/web3-tests.ts

```diff
@@ -90,6 +90,30 @@ myContract.options.from = "0x1234567890123456789012345678901234567891";
 myContract.options.gasPrice = "20000000000000";
 myContract.options.gas = 5000000;
 
+const newContract = new web3.eth.Contract(
+    [],
+    contractAddress,
+    {
+        from: "0x1234567890123456789012345678901234567891",
+        gasPrice: "20000000000"
+    }
+);
+const new_contract: Promise<TransactionReceipt> = newContract.deploy({
```

Comment:
Why `newContract` and `new_contract`. What is the logic behind this? Should `new_contract` not rather be `deploymentReceipt` or so?